core/domain/topic_fetchers.py

```diff
@@ -227,6 +227,22 @@ def get_topic_by_name(topic_name):
     return topic
 
 
+def get_all_topics_with_skills():
+    """Returns a list of topics with linked skills.
+
+    Returns:
+        list(Topic|None). A list of topics with skills.
+    """
+    all_topic_models = topic_models.TopicModel.get_all()
+    topics_with_skills = []
+    for topic_model in all_topic_models:
+        if topic_model:
```

Comment:
is there any specific reason we are expecting `all_topic_models` to have none within?

Reply:
Since, we are not searching by ID, I don't think `get_all` returns a None value.